reservoir-backend.js

```diff
@@ -103,6 +115,9 @@ function ReservoirBackend(options) {
     self.dropCount = {};
     self.logCount = {};
     self.samplingDecision = null;
+
+    var keys = Object.keys(options);
```

Comment:
don't implement this gaurd. It will create frustration of larch vs debug-larch because they take different options.